RecoEcal/EgammaClusterProducers/python/multi5x5SuperClusters_cfi.py

```diff
@@ -40,7 +40,8 @@
 
 
 multi5x5SuperClustersUncleaned = multi5x5SuperClustersCleaned.clone(
-    endcapClusterProducer = cms.string('multi5x5BasicClustersUncleaned')
+    barrelClusterTag = 'multi5x5BasicClustersUncleaned:multi5x5BarrelBasicClustersCleaned',
```

Comment:
```PYTHON
    barrelClusterTag= 'multi5x5BasicClustersUncleaned:multi5x5BarrelBasicClusters'),
    endcapClusterTag= 'multi5x5BasicClustersUncleaned:multi5x5EndcapBasicClusters'),
```

Then, in the config above I think one should replace
```PYTHON
    barrelClusterTag = cms.InputTag("multi5x5BasicClusters","multi5x5BarrelBasicClustersCleaned"),
```
with
```PYTHON
    barrelClusterTag= 'multi5x5BasicClustersCleaned:multi5x5BarrelBasicClusters'),
```

I think that was another bug in the original config, which was not exposed because  `doBarrel = cms.bool(False),`

@rchatter @argiro could you please check and confirm?

Reply:
@jeongeun I've checked the event content of step2 in wf 140.53, and I confirm what I wrote above:
```
vector<reco::CaloCluster>             "multi5x5BasicClustersCleaned"   "multi5x5BarrelBasicClusters"   "reRECO"   
vector<reco::CaloCluster>             "multi5x5BasicClustersUncleaned"   "multi5x5BarrelBasicClusters"   "reRECO"   
vector<reco::CaloCluster>             "multi5x5BasicClustersCleaned"   "multi5x5EndcapBasicClusters"   "reRECO"   
vector<reco::CaloCluster>             "multi5x5BasicClustersUncleaned"   "multi5x5EndcapBasicClusters"   "reRECO"   
```

Therefore, in multi5x5SuperClustersUncleaned one should add
```PYTHON
    barrelClusterTag= 'multi5x5BasicClustersUncleaned:multi5x5BarrelBasicClusters'),
    endcapClusterTag= 'multi5x5BasicClustersUncleaned:multi5x5EndcapBasicClusters'),
```
while in multi5x5SuperClustersCleaned one should fix as
```PYTHON